frontend/public/components/utils/log-window.jsx

```diff
@@ -50,8 +51,13 @@ export class LogWindow extends React.PureComponent {
   }
 
   _handleScroll() {
-    // Stream is finished, take no action on scroll
-    if (this.props.status === STREAM_EOF) {
+    const horizontalScroll = this.state.prevScrollLeft !== this.scrollPane.scrollLeft;
+
+    // Stream is finished or horizontal scroll, take no action
+    if (this.props.status === STREAM_EOF || horizontalScroll) {
+      if (horizontalScroll) {
+        this.setState({prevScrollLeft: this.scrollPane.scrollLeft});
```

Comment:
Wondering what the performance impact of calling `setState` from a scroll handler is... Should this just be kept as a class member?

Reply:
The rule of thumb seems to be that if you don't use a value in `render` you shouldn't add it to state

Reply:
Yeah, I was thinking about that as well, will adjust

Reply:
fixed